packages/@atjson/document/test/test-source.ts

```diff
@@ -8,6 +8,7 @@ import Document, {
 
 export class Anchor extends InlineAnnotation<{
   href: string;
+  target?: string | null;
```

Comment:
is null a valid value for this? I think it's probably just either a string or undefined right?

Reply:
This is a test 😬